apps/toffee/base/kustomize.yaml

```diff
@@ -8,4 +8,5 @@ spec:
   postBuild:
     substitute:
       NAMESPACE: "toffee"
+      WI_NAME: "toffee"
```

Comment:
looks unused? and probably unneeded? I think this is only for when WI_NAME doesn't match the namespace
```suggestion
```

Reply:
its being used here - https://github.com/hmcts/sds-flux-config/blob/65d77d56fa6913de86201aeb5479b166e2cbdf24/apps/base/workload-identity/workload-identity-ua-identity.yaml#L4 
I will change it to `NAMESPACE` variable instead